src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -48,11 +45,13 @@
       + "watch\\?v%3D|watch\\?feature=player_embedded&v=|%2Fvideos%2F|"
       + "embed%\u200C\u200B2F|youtu.be%2F|%2Fv%2F)[^#\\&\\?\\n]*";
 
+  @VisibleForTesting static final String PARAM_ID = "id";
```

Comment:
I left a comment about something similar to this in a different PR.
https://github.com/googleinterns/zoomtube/pull/187#discussion_r469553483

Reply:
Will refactor to make this more logical in #187

Reply:
sounds good!